pages/conference/young-inspirers.vue

```diff
@@ -0,0 +1,318 @@
+                        '提供我自己的一些心得、反思，以及給現代高中生的一些鼓勵和建議，' +
+                        '希望能鼓勵更多台灣的學生更加有興趣挑戰實習～' +
+                        '或許你會好奇高中生在較少專業領域的訓練下，如何爭取科技公司的實習機會？' +
+                        '或許你也會好奇高中生在實習中可以做些甚麼，以及會有什麼蛻變和優勢呢？' +
+                        '無論是對上述問題感到好奇、充滿疑問的你、或是正打算踏出實習的第一步的你，' +
+                        '希望我的故事能夠帶給你一些啟發，讓你也能成功打造出自己獨特的實習體驗～',
+                },
+                {
+                    title: '透過社團與科技的近距離接觸',
+                    speakers: [
+                        {
+                            name: '李米蕙',
+                            bio:
+                                '台灣師範大學大一新生，高中為北一女 AI 研究社第二屆副社長，' +
+                                '參訪過 Pinkoi、AWS、資安新創奧義智慧科技三家公司，' +
+                                '於 2021 年參加過第一屆 GiCS 尋找資安女婕思，' +
+                                '與隊友奪得高中職組創意發想賽第一名的佳績。',
+                            photo: require('~/static/img/young-inspirers/Li-Mi-Hui.png'),
+                        },
+                        {
+                            name: '蔡仲淇',
+                            bio:
+                                '目前為高三生，曾任北一 AI 研究社第三屆社長，' +
+                                '同為第一屆 GiCS 創意發想賽 高中職組第一名團隊的成員。',
+                            photo: require('~/static/img/young-inspirers/Tsai-Chung-Chi.jpg'),
+                        },
+                    ],
+                    description:
+                        '在這次的演講中，希望和大家分享我們在AI研究社的經驗與收穫，' +
+                        '其中包含選社的動機、學到的課程、公司的參訪體驗、' +
+                        '舉辦的社會服務活動與衍生的競賽過程。' +
+                        '提供我們透過社團與科技的接觸、學習到的知識及獲得的機會與經驗，' +
+                        '讓大家更了解有哪些方法可以藉由社團更進一步的增進自我。' +
+                        '許多學校應該都有資訊類的社團，' +
+                        '但是將焦點放在「人工智慧」的社團似乎還是比較少，' +
+                        '高中生能如何接觸這類資訊？又能從社團中學到什麼？' +
+                        '我們希望藉由今天的 30 分鐘和各位一起探討這些問題。',
+                },
+            ],
+        }
+    },
+    computed: {
+        bannerStyle() {
+            return {
+                'background-image': `url(${this.bannerUrl})`,
+                'background-repeat': 'no-repeat',
+                'background-size': 'contain',
+                'background-position': 'center',
+            }
+        },
+    },
+    head() {
+        return {
+            title: this.title,
+            meta: [
+                {
+                    hid: 'og:title',
+                    property: 'og:title',
+                    content: this.title,
+                },
+                {
+                    hid: 'og:description',
+                    property: 'og:description',
+                    content: this.intro,
+                },
+                {
+                    hid: 'description',
+                    name: 'description',
+                    content: this.intro,
+                },
+            ],
+        }
+    },
+}
+</script>
+
+<style lang="postcss" scoped>
+.intro {
+    @apply text-xs md:text-sm font-serif mb-8;
```

Comment:
`intro` should be `font-sans-serif`?

Reply:
I assume [`font-sans`](https://tailwindcss.com/docs/font-family#sans-serif) is for `sans-serif`?

Reply:
Yeah. I forgot Tailwind's class name for sans-serif.